### PR TITLE
*: adjust region heartbeat metrics.

### PR DESCRIPTION
**Pull request by disksing**
1. Record store id.
2. Increase counter after receive message.

### CODE REVIEW

File: server/grpc_service.go [hunk 1]
```diff
@@ -303,10 +303,14 @@ func (s *Server) RegionHeartbeat(stream pdpb.PD_RegionHeartbeatServer) error {
 			return errors.Trace(err)
 		}
 
+		storeID := request.GetLeader().GetStoreId()
+		storeLabel := fmt.Sprintf("tikv%d", storeID)
```
Comment (siddontang) [inline server/grpc_service.go]: I think you can remove tikv here.

Comment (disksing) [reply server/grpc_service.go]: I think `tikv42` is more meaningful than `42` when it shows on grafana page.

Comment (siddontang) [reply server/grpc_service.go]: We can configure how to show the name in Grafana directly, like `tikv-{store}`.

Comment (disksing) [reply server/grpc_service.go]: ok.

**Comment by siddontang** [comment]
LGTM

PTAL @huachaohuang

**Pull request merged**